### PR TITLE
LG-1457: saml redirect uris

### PR DESCRIPTION
**Pull request by solipet**
Allow Service Providers using the SAML protocol to specify multiple redirect uris to be included in the SAML response CSP headers.

### CODE REVIEW

**Comment by solipet** [comment]
That's what I get for fixing all rubocop complaints... 3 code climate errors. The nil checks are part of a Policy that check if a user is an admin, so I think those are fair.  How to mark errors as `ignore` or `won't fix`?

File: app/controllers/groups_controller.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 class GroupsController < ApplicationController
   before_action -> { authorize Group }
-  before_action :find_group, only: %i(show edit update destroy)
+  before_action :find_group, only: %i[show edit update destroy]
```
Comment (amoose) [inline app/controllers/groups_controller.rb]: group view action pending ;)

File: app/views/service_providers/_form.html.erb [hunk 2]
```diff
@@ -33,10 +33,16 @@
...
 
 <div class="oidc-fields">
-  <%= form.input :redirect_uris, label: "<b>Redirect URIs</b> — <i>OpenID Connect only</i><br>One or more URIs that login.gov will redirect to after authentication. These can be web URLs (public, internal, or localhost) or a custom scheme to support native applications, for example: <code>gov.agency.app://result</code>".html_safe %>
-  <button id="add-redirect-uri-field" class="usa-button" type="button">Add another URI</button>
+  <%= form.label :redirect_uris, "<b>Redirect URIs</b> — <i>OpenID Connect only</i><br>One or more URIs that login.gov will redirect to after authentication. These can be web URLs (public, internal, or localhost) or a custom scheme to support native applications, for example: <code>x-example-app:/result</code>".html_safe %>
 </div>
 
+<div class="saml-fields">
+  <%= form.label :redirect_uris, "<b>Redirect URIs</b> — <i>SAML only</i><br>One or more URIs that login.gov will include in the SAML response CSP headers to allow multiple targets for redirection. These can be web URLs (public, internal, or localhost) or a custom scheme to support native applications, for example: <code>x-example-app:/result</code>".html_safe %>
```
Comment (amoose) [inline app/views/service_providers/_form.html.erb]: I'd like the label to be "_Additional_ Redirect URIs" 

Also, these hints `x-example-app:/result` should be `gov.agency.app://result`.

Comment (amoose) [reply app/views/service_providers/_form.html.erb]: Remove "OpenID Connect only" and "SAML only" from the label, too.

File: app/views/service_providers/show.html.erb [hunk 3]
```diff
@@ -69,12 +69,10 @@
       <td><%= service_provider.return_to_sp_url %></td>
     </tr>
 
-    <% if service_provider.identity_protocol == 'openid_connect' %>
     <tr>
       <th scope="row">Redirect URIs</th>
```
Comment (amoose) [inline app/views/service_providers/show.html.erb]: label should match the form: "_Additional_ Redirect URIs"

**Pull request merged**